rotkehlchen/data_import/importers/rotki_trades.py

```diff
@@ -37,30 +34,39 @@ def _consume_rotki_trades(
         - DeserializationError
         - UnknownAsset
         - KeyError
-        - DivisionByZero if `amount_bought` is ZERO
         """
         amount_sold = deserialize_asset_amount(csv_row['Sell Amount'])
         amount_bought = deserialize_asset_amount(csv_row['Buy Amount'])
-        asset, fee, fee_currency, location, timestamp = process_rotki_generic_import_csv_fields(csv_row, 'Base Currency')  # noqa: E501
-        if (trade_type := TradeType.deserialize(csv_row['Type'])) == TradeType.BUY:
-            rate = amount_sold / amount_bought
-            amount = amount_bought
+        base_asset, fee, fee_currency, location, timestamp = process_rotki_generic_import_csv_fields(csv_row, 'Base Currency')  # noqa: E501
+        quote_asset = symbol_to_asset_or_token(csv_row['Quote Currency'])
+
+        # For a buy: spend quote asset, receive base asset
+        # For a sell: spend base asset, receive quote asset
+        if csv_row['Type'].strip().lower() == 'buy':
```

Comment:
why not use the deserialize as it was before? It's safer than using hardcoded strings and also easier to maintain

Reply:
because `TradeType.deserialize()` will go away with all the trade structure.